posthog/api/cohort.py

```diff
@@ -278,6 +278,9 @@ def validate_filters(self, raw: dict):
         1. structural/schema check → pydantic
         2. domain rules (feature-flag gotchas) → bespoke fn
         """
+        # Skip validation for static cohorts
+        if self.initial_data.get("is_static") or getattr(self.instance, "is_static", False):
+            return raw
```

Comment:
I noticed that for static cohorts, sometimes the filters look like `{"properties": {}}` and most of the time they're null. We probably should just set them to null every time, right?

Reply:
yeah probably, although static cohort filters don't really matter since we evaluate membership by looking up IDs in the cohortperson table.